### PR TITLE
make cluster.StressSpec pass with Artery, #21458

### PR DESCRIPTION
**Pull request by patriknw**
- need to use a shared media driver to get the cpu usage
  at a reasonable level
- also changed to SleepingIdleStrategy(1 ms) when cpu-level=1
  not needed for the test to pass, but can be good to make level 1
  more extreme

This is how it looks on my local now when running the StressSpec:
screen shot 2016-09-14 at 09 48 31

### CODE REVIEW

**Comment by patriknw** [comment]
Refs #21458

**Comment by patriknw** [comment]
There is still some debug println in here, but before cleaning up I wanted to see what jenkins says. I'll point the cluster job to this branch.

**Comment by patriknw** [comment]
That was a new failure: https://github.com/akka/akka/issues/21463

**Comment by patriknw** [comment]
This seems to work fine. Many successful runs, no failures apart from the known segfaults. https://jenkins.akka.io:8498/job/akka-artery-cluster-tests/

Please review, and then I will cleanup the println

**Comment by drewhk** [comment]
LGTM

File: akka-cluster/src/multi-jvm/scala/akka/cluster/StressSpec.scala [hunk 1]
```diff
@@ -699,8 +706,11 @@ class StressMultiJvmNode12 extends StressSpec
 class StressMultiJvmNode13 extends StressSpec
 
 abstract class StressSpec
-  extends MultiNodeSpec(StressMultiJvmSpec)
-  with MultiNodeClusterSpec with BeforeAndAfterEach with ImplicitSender {
+  extends MultiNodeSpec({
+    // Aeron media driver must be started before ActorSystem
+    SharedMediaDriverSupport.startMediaDriver(StressMultiJvmSpec)
+    StressMultiJvmSpec
+  }) with MultiNodeClusterSpec with BeforeAndAfterEach with ImplicitSender {
```
Comment (johanandren) [inline akka-cluster/src/multi-jvm/scala/akka/cluster/StressSpec.scala]: Is it impossible to provide some nicer hooks for this in the MultiNodeSpec?

**Review by johanandren — APPROVED** [review]

File: akka-cluster/src/multi-jvm/scala/akka/cluster/StressSpec.scala (hunk 1 above)
Comment (patriknw) [inline akka-cluster/src/multi-jvm/scala/akka/cluster/StressSpec.scala]: that's difficult, it starts the ActorSystem in it's constructor

**Comment by patriknw** [comment]
I cleaned up the printlns before merging

**Pull request merged**